.claude/skills/storybook/SKILL.md

```
@@ -0,0 +1,697 @@
+```
+
+### Interactive Stories (Play Functions)
+
+```typescript
+import { within, userEvent, expect } from "@storybook/test";
+
+export const ClickInteraction: Story = {
+  args: { children: "Click me" },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    const button = canvas.getByRole("button");
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)